ui/admin/app/components/form/credential-store/vault/index.hbs

```diff
@@ -0,0 +1,292 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @error={{@model.errors.base}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+  {{#if (feature-flag 'static-credentials')}}
+    <form.fieldset>
+      <:title>{{t 'form.type.label'}}</:title>
+      <:body>
+        {{#if @model.isNew}}
+          <form.radioGroup
+            @name='types'
+            @selectedValue={{@model.type}}
+            @changed={{@changeType}}
+            as |radioGroup|
+          >
+            <radioGroup.card
+              @label={{t 'resources.credential-store.types.static'}}
+              @helperText={{t 'resources.credential-store.help.static'}}
+              @value='static'
+              @layout='wide'
+            />
+            <radioGroup.card
+              @label={{t 'resources.credential-store.types.vault'}}
+              @helperText={{t 'resources.credential-store.help.vault'}}
+              @value='vault'
+              @layout='wide'
+            />
+          </form.radioGroup>
+        {{else}}
+          <form.radioGroup
+            @name='types'
+            @selectedValue={{@model.type}}
+            as |radioGroup|
+          >
+            <radioGroup.card
+              @label={{t
+                (concat 'resources.credential-store.types.' @model.type)
+              }}
+              @helperText={{t
+                (concat 'resources.credential-store.help.' @model.type)
+              }}
+              @value={{@model.type}}
+              @layout='wide'
+              readonly={{true}}
+              @disabled={{true}}
+            />
+          </form.radioGroup>
+        {{/if}}
+      </:body>
+    </form.fieldset>
+  {{else}}
+    <form.input
+      @value={{@model.type}}
+      @label={{t 'form.type.label'}}
+      readonly={{true}}
+      @disabled={{true}}
+    />
```

Comment:
Can this code be updated to use the radio card? That should be fine to display even if static credential work is not enabled yet.